custom/ilsgateway/models.py

```diff
@@ -136,7 +136,7 @@ class SupplyPointStatus(models.Model):
                                    max_length=50)
     status_value = models.CharField(max_length=50,
                                     choices=((c, c) for c in SupplyPointStatusValues.CHOICES))
-    status_date = models.DateTimeField(default=datetime.utcnow)
+    status_date = models.DateTimeField(default=datetime.utcnow, db_index=True)
     location_id = models.CharField(max_length=100, db_index=True)
```

Comment:
@emord is it worth it to take this opportunity to also add a FK reference to `sql_location` here for https://github.com/dimagi/commcare-hq/blob/master/custom/ilsgateway/tanzania/reminders/reports.py#L36-L42 ?